res/res_pjsip/location.c

```diff
@@ -1135,6 +1137,97 @@ static int cli_contact_print_body(void *obj, void *arg, int flags)
 		ast_sip_get_contact_short_status_label(status ? status->status : UNKNOWN),
 		(status && (status->status == AVAILABLE)) ? ((long long) status->rtt) / 1000.0 : NAN);
 
+	if (flags & AST_RETRIEVE_FLAG_ALL) { /*  If we are showing list, no need do detailed */
+		ao2_cleanup(status);
+		return 0;
+	}
+	/* Detailed view  or one entity start here */
```

Comment:
This should use the same functionality as the other CLI commands for printing out the detailed information, ast_sip_cli_print_sorcery_objectset - an example can be seen in the AOR command. This ensures that if Contact is extended with additional fields they will be automatically shown in the CLI command and also gives consistency across the various CLI commands.